definitions/npm/react-select_v1.x.x/flow_v0.104.x-/react-select_v1.x.x.js

```diff
@@ -168,7 +168,9 @@ declare module 'react-select' {
     valueRenderer?: (option: OptionType, idx?: number) => React$Node,
     // optional style to apply to the component wrapper
     wrapperStyle?: {...},
-    isSearchable?: boolean
+    isSearchable?: boolean,
+    // whether to cache the search results
+    cache?: boolean
```

Comment:
Shouldn't it be `void | false` instead?

Reply:
Sorry for the delay.

I'm relatively new to Flow, so I'm not sure.
I test it, and `void | false` cause it to not accept a `boolean` variable. If we want to have the `void` there, should it be `void | boolean`?

Reply:
The thing is from react-select doc it can only be false, if I'm not mistaken. 
But I'm probably nitpicking here.